### ISSUE TITLE
Documentation: Consider a better name for "emulate scrolling in fullscreen buffers" && Add it to README

### ISSUE THREAD

**Issue by NHDaly** [NHDaly/tmux-scroll-copy-mode#16]
From #15: `@prevent-scroll-for-fullscreen-alternate-buffer` now actually _enables scroll_ for those buffers, by emulating it.

Find a better name for this option.

**Comment by NHDaly** [comment]
My current best idea is `@emulate-scroll-for-no-mouse-alternate-buffer`.

**Comment by giddie** [comment]
Sounds OK to me :) I wonder if it might be worth enabling it by default? I suspect that the majority of people installing the plugin will want scrolling to "just work" in tmux, rather than compatibility with old tmux versions. Not a big deal, though, since the README isn't long.

**Comment by NHDaly** [comment]
> I wonder if it might be worth enabling it by default?

Yeah, I agree. But up until now, the default options have all been set to emulate the pre-2.1 behavior.

I've been considering changing the defaults to just be the best/most natural behavior, but I don't want to break people's configurations... I'm not sure how to make that change?

**Comment by NHDaly** [comment]
Fixed in 5e4c864.

**Comment by giddie** [comment]
I'm not sure most would consider it a break. If they've installed the plugin and left everything at default behaviour, it's probably because they're not too fussed about the behaviour, and if anything will probably be in for a pleasant surprise on upgrade. Those that specifically want compatibility with old tmux behaviour (do we know these people exist?) will probably be fussy enough that they'll notice the change immediately and refer to the README to modify their config.
